src/components/Calculator.js

```diff
@@ -20,13 +25,27 @@ const Calculator = () => {
   const curr = next || total || '0';
   const currOperation = getCurrOperation([total, operation, next, equalSign]);
   return (
-    <div>
-      <div className="Calculator">
-        <Screen value={[currOperation]} />
-        <Screen value={curr} />
-        <Buttons handleClick={handleClick} />
-      </div>
-    </div>
+    <>
+      <Navbar />
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/quotes">
+          <Quotes />
+        </Route>
+        <Route path="/calculator">
+          <div className="calculator-wrapper">
+            <Sidecalculator />
+            <div className="Calculator">
+              <Screen value={[currOperation]} />
+              <Screen value={curr} />
+              <Buttons handleClick={handleClick} />
+            </div>
+          </div>
+        </Route>
+      </Switch>
```

Comment:
[OPTIONAL]

- I would recommend putting all the `switch` and `route` in the `App.js` as the app component is parent one and in the complex structure, it's better to call the route and switch from the parent component rather than child one.

Reply:
Thank you